UnityProject/Assets/Scripts/Objects/Doors/DoorMasterController.cs

```diff
@@ -263,13 +271,19 @@ public void ClosedInteraction(HandApply interaction)
 		public void TryOpen(GameObject originator, bool blockClosing = false)
 		{
 			if(IsClosed == false || isPerformingAction) return;
-			
+
 			if(HasPower == false)
 			{
 				Chat.AddExamineMsgFromServer(originator, $"{gameObject.ExpensiveName()} is unpowered");
 				return;
 			}
 
+			if (isElectrecuted)
```

Comment:
this stuff should be put in DoorModuleBase

Reply:
But it doesn't extend from that?

Reply:
it should be its own module is what bod is saying

Reply:
ah, will look into that.